uiowa/uiowa
# Performance cleanup

Related to #7683

# How to test
- For each of the standard list block types, ensure that limit, limit per page, and paging is still working as expected and hasn't changed for existing blocks.
- Ensure that breadcrumb and caption are still displaying for featured images when they are displayed on a full node.

## Query logging
```
ddev general_log
```
Visit a page locally or run a command (in a different terminal window) and observe the overwhelming amount of query logs. Cmd-c to stop logging.
